packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.80.2",
+  "version": "3.80.3",
```

Comment:
```suggestion
  "version": "3.81.0",
```

Reply:
since we are adding a new prop, it should be a minor version change. not a patch change.